src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -107,9 +116,142 @@ public static void Profile(string assemblyFileName, string sessionName, Action a
 
             TraceEventSession.MergeInPlace(userFullFileName, Console.Out);
             WriteInfoLine($"ETW Tracing Session saved to \"{userFullFileName}\"");
+            collectOutputFilesCallback(userFullFileName);
+
+            var assemblyModel = GetAssemblyModel(assemblyFileName, userFullFileName, sessionName, performanceTestMessages);
```

Comment:
Ok, no worries.  Just want to make sure that we didn't put any xunit specifics in here.

---
In reply to: [102335366](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r102335366) [](ancestors = 102335366)